docs/reuse/apps/app-install.md

```diff
@@ -1,16 +1,20 @@
 To install the app:
 
-1. Click **App Catalog**, then search for and select the app.
-   * If you want to see a preview of the dashboards included with the app before installing, click **Preview Dashboards**.
-1. Select the version of the service you're using and click **Add to Library**.
-1. To install the app, complete the following fields:
-   * **App Name**. You can retain the existing name, or enter a name of your choice for the app.
-   * **Data Source**. Select either of these options for the data source.
-      * Choose **Source Category**, and select a source category from the list.
-      * Choose **Enter a Custom Data Filter**, and [enter a custom source category](/docs/get-started/apps-integrations#custom-data-filters) beginning with an underscore. Example: `_sourceCategory=MyCategory`.
-   * **Advanced**. Select the **Location in Library** (the default is the **Personal** folder in the library), or click **New Folder** to add a new folder.
-1. Click **Add to Library**.
+1. From the left nav, select **App Catalog**.<br/>  ![App_Catalog.png](/img/get-started/library/App-Catalog.png)
+1. Search for your app name and select it.
+1. When you get to your app page:
+   * If you want to see a preview of the dashboards included with the app before installing, scroll down to **Dashboard Preview**.
+   * If your Sumo Logic app has multiple versions (not all apps do), select the version of the service you're using.<br/><img src="/img/get-started/library/add-version.png" alt="add-version.png" width="30%" />
+   * Click **Add Integration**.<br/><img src="/img/get-started/library/add-integration.png" alt="add-integration.png" width="30%" />
+1. On the next configuration page, **Select Data Source for your App**, complete the following fields:
+   * **Data Source**. Select one of the following options:
+      * Choose **Source Category**, and select a source category from the list; or
+      * Choose **Enter a Custom Data Filter**, and [enter a custom source category](/docs/get-started/apps-integrations#custom-data-filters) beginning with an underscore. For example, `_sourceCategory=MyCategory`.
+   * **Folder Name**. You can retain the existing name or enter a custom name of your choice for the app.
+   * **All Folders** (optional). Default location is the **Personal** folder in your **Library**. If desired, you can choose a different location and/or click **New Folder** to add it to a new folder.
+1. Click **Next**.<br/><img src="/img/get-started/library/configure-next.png" alt="next button" width="80%" />
```

Comment:
The images in these last two steps could use a border.

Reply:
https://github.com/SumoLogic/sumologic-documentation/pull/2485/files#r1206986804